### ISSUE TITLE
learning rate strategy with different gradient accumulation

### ISSUE THREAD

**Issue by GANWANSHUI** [aharley/bev_baseline#8]
Hi, thanks again for the nice work!

With larger or lesser gradient accumulation， does it necessary to adjust the learning rate such as increasing and decreasing, respectively? Thanks again!

**Comment by aharley** [comment]
If you try values far away from ones mentioned in the readme, you may want to experiment a bit to optimize results. In general, I think the guideline is: with larger batch size you can use a larger learning rate.

**Comment by GANWANSHUI** [comment]
Thanks for the guidance, I will try it!
